src/main/kotlin/app/revanced/utils/adb/Commands.kt

```diff
@@ -2,12 +2,13 @@ package app.revanced.utils.adb
 
 import se.vidstige.jadb.JadbDevice
 import se.vidstige.jadb.RemoteFile
+import se.vidstige.jadb.ShellProcess
 import se.vidstige.jadb.ShellProcessBuilder
 import java.io.File
 
 internal fun JadbDevice.buildCommand(command: String, su: Boolean = true): ShellProcessBuilder {
     if (su) {
-        return shellProcessBuilder("su -c \'$command\'")
+        return shellProcessBuilder("su -mm -c \'$command\'")
```

Comment:
I couldn't find any documentation on `-mm`. What does it do?

Reply:
This is the global mount namespace thing, [look here.](https://android.stackexchange.com/a/200449)

It allows you to mount the application even if that setting is disabled in Magisk/SuperSU settings, (Magisk has it turned on by default but SuperSU has it off). But this is obviously useful even if the user has specifically turned it off as it is not globally excluding it but only for the specific thing.